gf-guide/metrics.md

```diff
@@ -232,7 +235,7 @@ The Typo Metrics need to inherit the v1.000 Win values. The WinAscent and WinDes
 2.  Repeat process for each weight/style if values are not unique in v1.000
 3.  Enable Use_Typo_Metrics
 
-If the font was previously hosted on [fonts.google.com](http://fonts.google.com/), you can test the upgraded vertical metrics visually match by using [GF Regression](https://github.com/googlefonts/gfregression). You can also use gftools gen-html with the command `gftools gen-html diff --font-before font1.ttf --font-after font2.ttf` if you want to compare two fonts locally.
+If the font is already hosted on [fonts.google.com](http://fonts.google.com/), you can test the upgraded vertical metrics visually match by using [diffenator2](https://github.com/googlefonts/diffenator2) with the command `diffenator2 diff -fb font1.ttf -fa font2.ttf -o out_dir` where `-fb` stands for `--fonts-before` and `-fa` for `--fonts-after`.
```

Comment:
We can add this good tip for GlyphsApp; in font info, parameter "EditView Line Height" which you can set to UPM*1.3 (so 1300 if your UPM value is 1000). This allows to view your linespacing within GlyphsApp.